_posts/2017/2017-03-30-finding-the-best-vindinium-bot-written-in-elixir.md

```diff
@@ -0,0 +1,51 @@
+Motivated to immerse in the world of [Elixir](http://elixir-lang.org/), he thought it couldn't be too difficult to write his own blogengine.
+And he was absolutely right!
+
+His blogengine Jelly Shot uses the web-framework [Phoenix](http://www.phoenixframework.org/) and is written in [Elixir](http://elixir-lang.org/).
+Unlike other blog systems, Jelly Shot has no access to databases nor uses static file generation, but stores data in its memory.
+The agent administers all data and is able to filter, delete, or replace parts of it - and this at a much faster pace.
+
+Check out Erik's [repository](https://github.com/erikmueller/jelly_shot) for more information!
+
+## The grand hh.ey vindinium challenge tournament
+
+After a short break to tank up new energy with a few snacks and drinks, everyone was ready to join or watch the vindinium show.
+Vindinium is an Artificial Intelligence programming challenge, in which the players have to take control of a legendary hero fighting for the land of Vindinium using their favorite programming language.
+In this case, there were three competitors who programmed their heroes with the functional language Elixir and the forth player was a random bot.
+The participants got really engaged with the four legendary heroes slashing goblins and stealing gold mines.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/epagesdevs">@epagesdevs</a> <a href="https://twitter.com/elixirlang">@elixirlang</a> <a href="https://twitter.com/HamburgEx">@HamburgEx</a> <a href="https://twitter.com/Air2ick">@Air2ick</a> <a href="https://twitter.com/h4cc">@h4cc</a> <a href="https://twitter.com/hashtag/guys?src=hash">#guys</a> playing and having fun like <a href="https://twitter.com/hashtag/kids?src=hash">#kids</a> :-)</p>&mdash; harm (@netzfisch) <a href="https://twitter.com/netzfisch/status/846787780357341186">March 28, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Everyone watched the screen with excitement to see which of the legendary heroes would collect the most gold to win the game.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">And the winner iiiiis: <a href="https://twitter.com/h4cc">@h4cc</a> :) <a href="https://twitter.com/hashtag/elixirlang?src=hash">#elixirlang</a> <a href="https://twitter.com/hashtag/vindinium?src=hash">#vindinium</a> <a href="https://t.co/uHkPTDbhkb">pic.twitter.com/uHkPTDbhkb</a></p>&mdash; ePages Developers (@epagesdevs) <a href="https://twitter.com/epagesdevs/status/846788530403774467">March 28, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+The meetup itself was a great way to spark discussions on the participants' experiences and faced challenges with the functional language Elixir. Thanks again for the great organization!
```

Comment:
... experiences and to face challenges with the ...
I would replace the last sentence with something like: It was a great evening and we were happy to again host the Elixir Meetup Hamburg.